main.tf

```diff
@@ -0,0 +1,144 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.0"
+    }
+  }
+}
```

Comment:
pretty minor, but i believe the convention is to put the `required_providers` block into a `versions.tf`

https://github.com/terraform-google-modules/terraform-google-sql-db/blob/main/modules/mysql/versions.tf

Reply:
Got it, updated in 7b83a04